reposync/reposync.py

```diff
@@ -184,49 +184,98 @@ def update_web_pages(root_dir):
 
 
 def canonicalize_jenkins_xml(xml: str) -> str:
+    def _extract_plugin_name(plugin_name_and_maybe_version: str) -> str:
```

Comment:
This fixes a bug where airtables-utils thought that *all* Jenkins XML configurations needed to change since it couldn't correctly extract the plugin name where no `@version-number` (e.g. `git@4.13`) was present in the string.